slave.mk

```diff
@@ -1573,7 +1575,9 @@ SONIC_CLEAN_TARGETS += $(addsuffix -clean,$(addprefix $(TARGET_PATH)/, \
 		       $(SONIC_DOCKER_IMAGES) \
 		       $(SONIC_DOCKER_DBG_IMAGES) \
 		       $(SONIC_SIMPLE_DOCKER_IMAGES) \
-		       $(SONIC_INSTALLERS)))
+		       $(SONIC_INSTALLERS) \
+		       $(SONIC_RFS_TARGETS)))
```

Comment:
Can you add caching support for rootfs preparation in squashfs file system?
You can refer DPKG caching framework for deb packages.

Reply:
Added in https://github.com/sonic-net/sonic-buildimage/pull/15924/commits/a880d53d14d593f6739e0ca33d22354157503aec